engine/api/src/main/java/org/apache/cloudstack/engine/orchestration/service/VolumeOrchestrationService.java

```diff
@@ -112,7 +112,7 @@ DiskProfile allocateRawVolume(Type type, String name, DiskOffering offering, Lon
 
     void release(VirtualMachineProfile profile);
 
-    void release(long vmId, long hostId);
+    void release(VirtualMachine vm, Host host);
```

Comment:
I am not to sure about passing objects from service to service here. Is this change needed?

Reply:
@DaanHoogland removed the changes